### PR TITLE
 Remove ContractFeatures

### PR DESCRIPTION
**Pull request by chenzhitong**
sync with https://github.com/neo-project/neo/pull/2060

### CODE REVIEW

File: templates/Template.NEP5.CSharp/NEP5.cs [hunk 1]
```diff
@@ -10,7 +10,6 @@ namespace Template.NEP5.CSharp
     [ManifestExtra("Email", "dev@neo.org")]
     [ManifestExtra("Description", "This is a NEP5 example")]
     [SupportedStandards("NEP5", "NEP10")]
-    [Features(ContractFeatures.HasStorage | ContractFeatures.Payable)]
```
Comment (Tommo-L) [inline templates/Template.NEP5.CSharp/NEP5.cs:13]: Remove `Features`.

**Comment by chenzhitong** [comment]
Unit test run passed (Need to test in conjunction with https://github.com/neo-project/neo/pull/2060)

**Review by vncoelho — COMMENTED** [review]
Very good, @chenzhitong

**Review by shargon — APPROVED** [review]

**Comment by shargon** [comment]
Merge?

**Pull request merged**